crates/blockchain-tree/src/chain.rs

```diff
@@ -291,16 +294,17 @@ impl AppendableChain {
             canonical_fork,
         };
 
-        let (block_state, _) = Self::validate_and_execute(
+        let (block_state, trie_updates) = Self::validate_and_execute(
             block.clone(),
             parent_block,
             bundle_state_data,
             externals,
             block_attachment,
             block_validation_kind,
+            self.chain.trie_updates().map(|u| u.sorted()),
```

Comment:
wonder if we can avoid the clone here somehow, maybe with `self.chain.trie_updates_mut().map(|u| u.sort_in_place())`?

would need a new function here https://github.com/paradigmxyz/reth/blob/23fc18db251381b79a17c122e28f49d6a7a6436a/crates/evm/execution-types/src/chain.rs#L82

Reply:
we want a vector for sorted updates which would need clone anyway.